velox/docs/functions/presto/conversion.rst

```diff
@@ -741,6 +742,7 @@ Valid example
   SELECT cast(0.12 as decimal(4, 1)); -- decimal '0.1'
   SELECT cast(0.19 as decimal(4, 1)); -- decimal '0.2'
   SELECT cast(0.123456789123123 as decimal(38, 18)); -- decimal '0.123456789123123000'
+  SELECT cast(cast(0.123456 as real) as decimal(38, 18)); -- decimal '0.123456000000000000'
```

Comment:
We can also write it as below

SELECT cast(real '0.123456' as decimal(38, 18))

Reply:
Thanks for your review. Updated.